src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -98,4 +108,27 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Redirect back to the HTML page.
     response.sendRedirect("/index.html");
   }
+
+  /** Identifies the sentiment in the journal text entry string. */
```

Comment:
Not a big deal, but keep consistency of code by using // for single line comments

Reply:
Done :)